src/Microsoft.SqlTools.ServiceLayer/AzureFunctions/AzureFunctionsService.cs

```diff
@@ -33,6 +33,7 @@ public static AzureFunctionsService Instance
         public void InitializeService(ServiceHost serviceHost)
         {
             serviceHost.SetRequestHandler(AddSqlBindingRequest.Type, this.HandleAddSqlBindingRequest);
+            serviceHost.SetRequestHandler(GetAzureFunctionsRequest.Type, this.HandleGetAzureFunctionsRequest);
```

Comment:
Do I remember correctly that you mentioned in ADS the option shows only for .cs files - I am assuming we are checking file extns on ADS side itself?

Reply:
yup the command only shows in the command palette when `editorLangId == csharp`